tsconfig.json

```diff
@@ -1,13 +1,21 @@
 {
   "compilerOptions": {
     "outDir": "./dist/",
+    "rootDir": "./",
+    "target": "es5",
     "module": "commonjs",
+    "moduleResolution": "node",
```

Comment:
ok - these flags match up almost exactly with compassql - but we can remove most.